sql/step_81_drop_adm_deleted.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE oh_admission DROP COLUMN pat_deleted;
```

Comment:
Shouldn't the column that is being dropped be `adm_deleted`?  `pat_deleted` doesn't exist in the table.